test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala

```diff
@@ -74,6 +74,16 @@ trait CoreTestData {
   val importJourneyWithTwoCompleteGoodsEntries: DeclarationJourney =
     startedImportToGreatBritainJourney.copy(goodsEntries = completedDeclarationJourney.goodsEntries)
 
+  val importJourneyWithGoodsOverThreshold: DeclarationJourney =
+    startedImportToGreatBritainJourney.copy(
+      goodsEntries = GoodsEntries(completedGoodsEntry.copy(
+        maybePurchaseDetails = Some(PurchaseDetails("10000", Currency("Eurozone", "Euro", "EUR"))))))
```

Comment:
Probably better to set the to be just over the threshold.

Reply:
have changed to 1915 which is just over £1500 * our stubbed conversion rate of 1.2763

Reply:
Cool - might be even better to do this as a calculation in the spec to show the relationship, e.g.

val amountOverTheThreshold = 1501 * CurrencyConversionStub.euroConversionRate

and use that as the amount if you see what I mean.

but approved anyways...